wandb/sdk/wandb_run.py

```diff
@@ -2224,6 +2236,18 @@ def _on_finish(self) -> None:
         if self._run_status_checker:
             self._run_status_checker.join()
 
+        self._unregister_telemetry_import_hooks(self._run_id)
+
+    @staticmethod
+    def _unregister_telemetry_import_hooks(run_id: str) -> None:
+        import_telemetry_set = {
```

Comment:
cant you use: `list_telemetry_imports()` function

Reply:
ah good catch! will fix!